src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -174,8 +257,14 @@ public int AddAdornments()
                     SetupView(secondaryView);
                 }
 
-                var enabled = _optionService.GetOption(NavigationBarOptions.ShowNavigationBar, _languageService.RoslynLanguageName);
-                AddOrRemoveDropdown(enabled);
+                ErrorHandler.ThrowOnFailure(_codeWindow.GetBuffer(out var buffer));
```

Comment:
This also makes me feel a bit happier this being deferred until AddAdornments, since by then the buffer probably exists whereas before it could have still been in some lazy state.